public/index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <title>Governo Italiano - Report Vaccini Anti Covid-19</title>
+    <title>Report Vaccini Anti Covid-19</title>
```

Comment:
```suggestion
    <title>Example Dashboard - Vaccini Anti Covid-19</title>
```

Let's differenciate this from the official one also in the naming